src/common/python/utility.py

```diff
@@ -84,3 +84,34 @@ def get_error_message(identifier, error):
 
 def get_aws_account_id():
     return boto3.client('sts').get_caller_identity().get('Account')
+
+
+def get_instance_snapshot_arn(snapshot_name, region):
+    """returns instance snapshot arn if in available state"""
+    rds = boto3.client('rds', region)
+    snapshots_response = rds.describe_db_snapshots(DBSnapshotIdentifier=snapshot_name)
+    assert snapshots_response['ResponseMetadata'][
+               'HTTPStatusCode'] == 200, f"Error fetching DB snapshots: {snapshots_response}"
+    snapshots = snapshots_response['DBSnapshots']
+    assert len(snapshots) == 1, f"No snapshot matches name {snapshot_name}"
+    snap = snapshots[0]
+    snap_status = snap.get('Status')
+    if snap_status == 'available':
+        return snap['DBSnapshotArn']
+    else:
+        raise Exception(f"Snapshot is not available yet, status is {snap_status}")
+
+
+def supports_snapshot_export_region(region):
+    rds = boto3.client('rds', region)
+    try:
+        rds.describe_export_tasks()
+    except Exception as error:
+        return "InvalidParameterCombination) when calling the DescribeExportTasks operation: This operation is not currently supported." not in str(error)
+    return True
+
+
+def eval_export_exception(export_snapshot_supported_region):
+    if export_snapshot_supported_region == "" or export_snapshot_supported_region.isspace():
```

Comment:
can use `if(not (str and not str.isspace()))` or similar constructs

Reply:
ok: will change str == "" to not str